webapp/package.json

```diff
@@ -34,6 +34,7 @@
     "dotenv": "^5.0.0",
     "electron-log": "^4.4.7",
     "electron-updater": "^5.0.1",
+    "express": "^4.21.1",
```

Comment:
Instead here, I would rather add 

```
    RUN npm install express
```

to the Dockerfile in webapp directory.

Reply:
Oh is that because you don't do express for electron builds or something?

Reply:
Yes exactly, and for the normal web build